src/lib/api/axiosInstanceRequest.test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file axiosInstanceRequest.test.js
+ * Contains tests for axiosInstance.js that involve mocking requests.
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import mockAxios from 'jest-mock-axios';
+
+import axiosInstance from './axiosInstance';
+import { API_ENDPOINT_XCSRF_TOKEN } from '../../constants';
+
+afterEach(() => mockAxios.reset());
+
+describe('api->axiosInstance()->{method}()', () => {
+  // TODO: Eventually, we need to add tests for axiosInstance's error handling.
+  // However, jest-mock-axios does not support interceptors at the moment. This
+  // means that, since our error handling is (correctly) done in an interceptor,
+  // we will need to fork jest-mock-axios and PR support for interceptors before
+  // we can test our error handling.
```

Comment:
So, I added an axios mock that implements jest-mock-axios. This allows for interceptors to be defined, but jest-mock-axios will never execute them. So, yeah, that needs work.